### PR TITLE
docs: Edits to dm theory vignette

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: vignettes/howto-dm-theory.Rmd [hunk 1]
```diff
@@ -81,24 +83,24 @@ flights %>%
...
-```{r warning=F, message=F}
+```{r warning=FALSE, message=FALSE}
 dm <- dm_nycflights13(cycle = TRUE)
 
 dm %>%
   dm_draw()
 ```
 
-The `flights` table is linked to three other tables: `airlines`, `planes` and `airports`.
+The `flights` table is linked to three other tables: `airlines`, `planes`, and `airports`. <!-- What about `weather` table? -->
```
Comment (krlmlr) [inline vignettes/howto-dm-theory.Rmd:103]: Good catch, this was before we had compound keys.

**Review by krlmlr — APPROVED** [review]
Thanks!

File: vignettes/howto-dm-theory.Rmd (hunk 1 above)
Comment (IndrajeetPatil) [reply vignettes/howto-dm-theory.Rmd]: Cool. I have added it to the list.

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**